src/Razor/src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/MvcImportProjectFeature.cs

```diff
@@ -22,7 +22,7 @@ public IReadOnlyList<RazorProjectItem> GetImports(RazorProjectItem projectItem)
             }
 
             // Don't add MVC imports for a component - this shouldn't happen for v1, but just in case.
-            if (FileKinds.IsComponent(projectItem.FileKind))
+            if (FileKinds.IsComponent(projectItem.FileKind) || FileKinds.IsComponentImport(projectItem.FileKind))
```

Comment:
Could we make it so that `FileKinds.IsComponent` returns true if it's FileKind = Components or ComponentsImport? After alll imports.razor are component files, no?

Reply:
Done